src/gui/DatabaseWidget.cpp

```diff
@@ -478,6 +478,28 @@ void DatabaseWidget::deleteGroup()
         return;
     }
 
+    if (recycleBinSelected()) {
+        if (recycleBinEmpty()) {
+            delete currentGroup;
+            return;
+        }
+
+        QMessageBox::StandardButton result = MessageBox::question(
```

Comment:
* Unnecessary .arg() call.
* Would be better to explicitly set QMessageBox::No as the default.